src/styles/blocks/bike.scss

```diff
@@ -0,0 +1,36 @@
+.bike {
+  grid-column: span 2;
+
+  @include onTablet {
+    grid-column: 2 / span 4;
+  }
+
+  @include onDesktop {
+    grid-column: span 4;
+  }
+
+  &__photo {
+    width: 100%;
+    margin-bottom: 32px;
+
+    @include hover(transform, scale(1.2));
+  }
+
+  &__product-title {
+    font-size: 24px;
+    line-height: 140%;
+    font-weight: 700;
+    margin-bottom: 8px;
+  }
+
+  &__description {
+    line-height: 140%;
+    margin-bottom: 16px;
+  }
+
+  &__price {
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/63805271/cf752891-fb72-40da-8ea4-d62f0841e4a5)
If you can, add please margin-top for the third price

Reply:
It looks the same as on the design in the figma)